### DISCUSSION TITLE
I am fustrated with plausible

### DISCUSSION THREAD

**Discussion by honey7988023** [plausible/analytics#4351] [Self-Hosted Support]
After hours of hardwork, I sucessfully set it up my self hosted plausible. However, Now it is not letting me create my account. whenever i click on create account button nothing happens. can someone give me proper solution of this as i am tired now.

**Comment by ruslandoga** [comment]
👋 @honey7988023 

Please check the browser console for WebSocket connection errors. More info: https://github.com/plausible/analytics/discussions/4310#discussioncomment-9957615

**Comment by ruslandoga** [comment]
What did you find hard or frustrating about Plausible installation? If you share the details, we will try to improve it :)

**Comment by honey7988023** [comment]


**Reply by honey7988023** [reply]
@ruslandoga

**Reply by ruslandoga** [reply]
You don't have WebSockets enabled in your nginx configuration :) Please see the discussion I linked above for more details.

**Reply by ruslandoga** [reply]
> tried to install plausible using online tutorials.

For a more straightforward guide you can check our quickstart at https://github.com/plausible/community-edition#quick-start

**Reply by honey7988023** [reply]
@ruslandoga

**Comment by honey7988023** [comment]


**Reply by ruslandoga** [reply]
Please make sure to restart nginx for the new config to take effect. `systemctl restart nginx` might work.

**Comment by honey7988023** [comment]


**Reply by ruslandoga** [reply]
While we appreciate your interest and enthusiasm, I kindly request that you refrain from tagging me unless it's absolutely necessary. Tagging repeatedly can be disruptive and may not be the best approach to get the assistance or attention you need.

**Comment by honey7988023** [comment]
Sorry brother, i admit my mistake
